src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpTraceExportProcessorBuilder.cs

```diff
@@ -0,0 +1,111 @@
+// <copyright file="OtlpTraceExportProcessorBuilder.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#nullable enable
+
+using System.Diagnostics;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation;
+using OpenTelemetry.Internal;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Exporter.OpenTelemetryProtocol;
+
+internal class OtlpTraceExportProcessorBuilder
+{
+    private readonly TracerProviderBuilder tracerProviderBuilder;
+    private readonly string name;
+
+    internal OtlpTraceExportProcessorBuilder(
+        TracerProviderBuilder tracerProviderBuilder,
+        string name,
+        IConfiguration? configuration)
+    {
+        Debug.Assert(tracerProviderBuilder != null, "tracerProviderBuilder was null");
+        Debug.Assert(name != null, "name was null");
+
+        this.tracerProviderBuilder = tracerProviderBuilder!;
+        this.name = name!;
+
+        if (configuration != null)
+        {
+            this.tracerProviderBuilder.ConfigureServices(services =>
+            {
+                services.AddOptions<OtlpTraceExporterOptions>(name).Bind(configuration);
+                services.AddOptions<ExportActivityProcessorOptions>(name).Bind(configuration.GetSection("ActivityProcessorOptions"));
+            });
+        }
+    }
+
+    public OtlpTraceExportProcessorBuilder ConfigureExporterOptions(Action<OtlpTraceExporterOptions> configure)
+    {
+        Guard.ThrowIfNull(configure);
+
+        return this.ConfigureServices(services => services.Configure(this.name, configure));
+    }
+
+    public OtlpTraceExportProcessorBuilder ConfigureProcessorOptions(Action<ExportActivityProcessorOptions> configure)
+    {
+        Guard.ThrowIfNull(configure);
+
+        return this.ConfigureServices(services => services.Configure(this.name, configure));
+    }
+
+    public OtlpTraceExportProcessorBuilder ConfigureSdkLimitOptions(Action<SdkLimitOptions> configure)
```

Comment:
From the standpoint of the spec, these limit options are a global SDK configuration. This should probably become a configuration extension on `TracerProvider` and `LoggerProvider` rather than on the OTLP exporter builders. I think if exporters had a handle back to their parent provider then they should be able to get this limit configuration.